pom.xml

```diff
@@ -214,6 +214,8 @@
 
         <osgi.framework.imp.pkg.version.range>[1.7.0, 2.0.0)</osgi.framework.imp.pkg.version.range>
         <osgi.service.component.imp.pkg.version.range>[1.2.0, 2.0.0)</osgi.service.component.imp.pkg.version.range>
+        <commons-lang.wso2.osgi.version.range>[2.6.0,3.0.0)</commons-lang.wso2.osgi.version.range>
```

Comment:
Refer [1] in general. Use similar ordering and formatting.

[1] https://github.com/wso2-extensions/identity-event-handler-notification/blob/master/pom.xml